docs/architecture/adr-008-square-size-independent-message-commitments.md

```diff
@@ -104,14 +104,14 @@ We should note that Rollups can decide to do this scheme without changing core-a
 
 ## Positive celestia-app changes
 
-- Simplifies the creation of PFDs because users don't need to create commitments for multiple square sizes.
-  - Reduces the size of [MsgWirePayForData](https://github.com/celestiaorg/celestia-app/blob/6f3b3ae437b2a70d72ff6be2741abb8b5378caa0/x/payment/types/tx.pb.go#L32-L40)s because MessageShareCommitment can be modified from an array of maximum length 8 (for all valid square sizes) to a single MessageShareCommitment.
-  - Simplifies the malleation process because this ADR enables a future refactor to remove WireMsgPayForData entirely ([issue](https://github.com/celestiaorg/celestia-app/issues/951)). Previously multiple signatures were included in a WireMsgPayForData and only one was used to construct the MsgPayForData that ended up on-chain but this ADR results in only one signature needed on the wrapping SDK message and the SDK message that ends up on-chain.
+- Simplifies the creation of PFBs because users don't need to create commitments for multiple square sizes.
+  - Reduces the size of [MsgWirePayForBlob](https://github.com/celestiaorg/celestia-app/blob/6f3b3ae437b2a70d72ff6be2741abb8b5378caa0/x/payment/types/tx.pb.go#L32-L40)s because MessageShareCommitment can be modified from an array of maximum length 8 (for all valid square sizes) to a single MessageShareCommitment.
```

Comment:
[no change needed]

we're in a chicken and egg situation because the permalink https://github.com/celestiaorg/celestia-app/blob/6f3b3ae437b2a70d72ff6be2741abb8b5378caa0/x/payment/types/tx.pb.go#L32-L40 still points to the old name. Super minor but after this PR merges, we can update the permalinks to reference the lines of code with the new name

Reply:
created #991